src/Avalonia.Themes.Fluent/Controls/TextBox.xaml

```diff
@@ -102,7 +114,7 @@
     <Setter Property="Padding" Value="{DynamicResource TextControlThemePadding}" />
     <Setter Property="FocusAdorner" Value="{x:Null}" />
     <Setter Property="ScrollViewer.IsScrollChainingEnabled" Value="True" />
-    <Setter Property="ContextFlyout" Value="{StaticResource DefaultTextBoxContextFlyout}" />
+    <Setter Property="ContextFlyout" Value="{OnFormFactor Desktop={StaticResource DefaultTextBoxContextFlyout}, Mobile={StaticResource MobileTextBoxContextFlyout}}" />
```

Comment:
If anything, it should differentiate on input type, but can be changed after media queries merged.

Reply:
This could be solved by the earlier  suggestion for a SelectionFlyout. I don't think media queries can differentiate by input.